types/LinkExitTypes.d.ts

```diff
@@ -0,0 +1,157 @@
+import { LinkInstitution } from "./LinkInstitution";
+
+export interface LinkExit {
+    error?: LinkError;
+    metadata: LinkExitMetadata;
+}
+
+export interface LinkExitMetadata {
+    link_session_id: string;
```

Comment:
Is our convention `snake_case` or `camelCase`?
This file contains a mix, whereas https://github.com/plaid/react-native-plaid-link-sdk/pull/234/files was all `camelCase`

Reply:
sessionId should also be optional

Reply:
yea, I'll move these to camelCase